Makefile

```diff
@@ -56,7 +73,7 @@ vendor: tools.timestamp vendor.conf
 	@touch vendor
 
 clean:
-	rm -rf vendor tools.timestamp
+	rm -rf vendor tools.timestamp docs/*.5
```

Comment:
`$(MANPAGES)` please, to minimize duplication.

Reply:
Fixed